datalake/src/main/java/com/sequenceiq/datalake/service/sdx/ProvisionerService.java

```diff
@@ -175,7 +176,7 @@ public void startStackProvisioning(Long id, DetailedEnvironmentResponse environm
                 sdxClusterRepository.save(sdxCluster);
                 cloudbreakFlowService.saveLastCloudbreakFlowChainId(sdxCluster, stackV4Response.getFlowIdentifier());
                 LOGGER.info("Sdx cluster updated");
-            } catch (ClientErrorException e) {
+            } catch (ClientErrorException | InternalServerErrorException e) {
```

Comment:
hmm.. can we throw badrequest instead of internal server error from cloudbreak side?

Reply:
We do not throw internal exception directly, It is a common jax-rs exception of HTTP code 500. But we need to handle it because the reason message was `500 internal error` on the UI

Reply:
but why is it 500? why not something else? maybe a bad exceptionmapper?

Reply:
Imagine that user error when someone would like to attach an AZURE SDX to an AWS env.
Somewhere in the CB throw a ClassCastException. Should we create a mapper all of the unexpected wrong user interaction? 
Sorry, my fault: in the `DefaultExceptionMapper` we throw an InternalServerError

Reply:
please use WebApplicationException (it will catch both client and server errors)